src/main/java/ch/njol/skript/doc/HTMLGenerator.java

```diff
@@ -488,7 +488,13 @@ private String generateAnnotated(String descTemp, SyntaxElementInfo<?> info, @Nu
 		desc = handleIf(desc, "${if by-addon}", false);
 
 		// New Elements
-		desc = handleIf(desc, "${if new-element}", NEW_TAG_PATTERN.matcher((since != null ? since.value() : "")).find());
+		if (since != null) {
+			for (String s : since.value()) {
```

Comment:
Hmm I think you can only call handleIf once for a tag, so you might need to check it any of the sinces match (or join them with like space to create one string  to run the matcher on)

Reply:
As discussed on discord, I changed this to only check the last element of the array.